src/workloads/filesystem/Flushing.yml

```diff
@@ -0,0 +1,21 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-perf"
+Description: >
+  Run large insertMany's to stress checkpointing, journal flush and dirty memory with {w:majority}.
+  The important ftdc metrics to look at are:
+    - ss opcounters insert
+    - ss average latency writes
+    - system memory dirty
+    - ss wt log log sync time duration
+    - ss wt transaction transaction checkpoint currently running
+
+LoadConfig:
+  Path: ../../phases/filesystem/FlushingTemplate.yml
```

Comment:
This is probably personal preference, but I would prefer having the workload definition in a single file and moving it into a separate file once we do actually need to reuse it. Feel free to ignore this one.